src/components/Stepper/StepperMobileExamples.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+const propTypes = {
+  tag: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+  className: PropTypes.string,
+  children: PropTypes.element,
+  dark: PropTypes.string
+}
+
+const defaultProps = {
+  tag: 'div'
+}
+
+const StepperMobileExamples = props => {
```

Comment:
To me the only difference with a regular `StepperContainer` is a CSS class.
Can we merge these and provide a `mobile` prop?